### DISCUSSION TITLE
reference/get_started/install

### DISCUSSION THREAD

**Discussion by giscus** [devlive-community/datacap#291] [Q&A] [answered]
# reference/get_started/install

DataCap is integrated software for data transformation, integration and visualization.

https://datacap.edurt.io/reference/get_started/install.html

**Comment by yuzhaoyong** [comment]
安装报错：2023-03-20 15:25:54,242 ERROR [http-nio-9096-exec-8] org.hibernate.engine.jdbc.spi.SqlExceptionHelper [SqlExceptionHelper.java:142] Unknown column 'userentity0_.third_configure' in 'field list'

**Reply by yuzhaoyong** [reply]
版本是datacap1.7

**Reply by qianmoQ** [reply]
需要执行一下1.7.0版本更新的SQL文件

**Comment by k9flame** [comment]
我安装1.8.0报错，错误: 找不到或无法加载主类 io.edurt.datacap.server.DataCap

**Reply by qianmoQ** [reply, marked as answer]
需要在项目根目录中运行 `./bin/startup.sh`

**Reply by k9flame** [reply]
好了，感谢感谢

**Comment by mc1020** [comment]
mongo 怎么获取元数据信息啊

**Reply by qianmoQ** [reply]
具体是指哪些呢？

**Reply by qianmoQ** [reply]
```
db.collectionName.stats()
db.getCollectionInfos()
db.collectionName.getIndexes()
db.system.namespaces.find()
```

**Reply by mc1020** [reply]
就是我添加了mogodb数据源，但是查询页面，左边树结构那里没有显示有哪些库，有哪些集合。如果我想想让他左边树结构出来，我应该再模板里面加什么语句啊。我试了sql不行，查不出来
